src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Jurisdiction.java

```diff
@@ -85,41 +84,30 @@
     )
     private YesOrNo bothLastHabituallyResident;
 
-    @CCD(
-        label = "Jurisdiction connections",
-        hint = "Tick all the reasons that apply:",
-        access = {DefaultAccess.class}
-    )
-    private Set<JurisdictionConnections> connections;
-
     @CCD(
         label = "Legal connections",
         hint = "Tick all the reasons that apply:",
         access = {DefaultAccess.class}
     )
-    private Set<LegalConnections> legalConnections;
```

Comment:
Would it be better to not delete the field at the moment and just mark it for deletion later.  There are a few other fields marked for deletion.